rtlib/include/rt/LandmarkIO.hpp

```diff
@@ -16,25 +17,16 @@ class LandmarkIO {
 
     void setFixedImage(const Image8UC3::Pointer i) { fixedImage_ = i; }
     void setMovingImage(const Image8UC3::Pointer i) { movingImage_ = i; }
-    void setLandmarksPath(const boost::filesystem::path& path)
-    {
-        landmarksPath_ = path;
-    }
+    void setLandmarksPath(const boost::filesystem::path& path) { landmarksPath_ = path; }
```

Comment:
Didn't realize that there was a 80 character limit. Will go back and run clang again.